tests/unit/test_bigquery_adapter.py

```diff
@@ -349,10 +349,10 @@ def test_get_relation(self):
         self.adapter.get_relation('db', 'schema', 'my_model')
         self.mock_connection_manager.get_bq_table.assert_called_once_with('db', 'schema', 'my_model')
 
-    def test_create_schema(self):
-        relation = BigQueryRelation.create(database='db', schema='schema')
-        self.adapter.create_schema(relation)
-        self.mock_connection_manager.create_dataset.assert_called_once_with('db', 'schema')
+#    def test_create_schema(self):
+#        relation = BigQueryRelation.create(database='db', schema='schema')
+#        self.adapter.create_schema(relation)
+#        self.mock_connection_manager.create_dataset.assert_called_once_with('db', 'schema')
```

Comment:
@jtcohen6 do we plan on leaving the test commented out or is it okay to remove?

Reply:
Good catch - probably okay to remove. I don't think we can have a dedicated unit test like this (mocking a Python method) if dbt is running real SQL instead. I also don't feel like we need a dedicated test for `create_schema`, since it's a required "setup" step in every single functional test.